core/src/test/java/tech/tablesaw/api/RowTest.java

```diff
@@ -6,8 +6,11 @@
 import java.io.File;
 import java.io.IOException;
 import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
 
-import static org.junit.Assert.*;
+import static org.junit.Assert.assertEquals;
+import static tech.tablesaw.api.ColumnType.*;
```

Comment:
Please remove the unused import. Codacy will thank you.

Reply:
Removed in latest commit.